example/openapi/api-docs.yaml

```diff
@@ -1035,331 +1412,419 @@ components:
+          readOnly: true
         _creator:
           type: string
+          readOnly: true
         _docId:
           type: string
-        _projectId:
-          type: string
+          readOnly: true
         _modifier:
           type: string
-        _modified:
-          type: string
+          readOnly: true
         _created:
           type: string
-        _commitId:
+          readOnly: true
+        _modified:
           type: string
-      additionalProperties:
-        type: object
-    CommitsResponse:
+          readOnly: true
+      additionalProperties: true
+    RefsResponse:
       type: object
       properties:
         messages:
           type: array
+          nullable: true
           items:
             type: string
+            nullable: true
         rejected:
           type: array
+          nullable: true
           items:
             $ref: '#/components/schemas/Rejection'
-        commits:
+        refs:
           type: array
           items:
-            $ref: '#/components/schemas/CommitJson'
-    CommitsRequest:
+            $ref: '#/components/schemas/Ref'
+    RefsRequest:
+      required:
+      - refs
       type: object
       properties:
-        source:
-          type: string
-        comment:
-          type: string
-        commits:
+        refs:
           type: array
           items:
-            $ref: '#/components/schemas/CommitJson'
-    ProjectJson:
+            $ref: '#/components/schemas/Ref'
+    Commit:
+      required:
+      - id
       type: object
       properties:
-        projectType:
-          type: string
-        orgId:
-          type: string
-        type:
+        deleted:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/ElementUpdated'
+        added:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/ElementUpdated'
+        updated:
+          type: array
+          readOnly: true
+          items:
+            $ref: '#/components/schemas/ElementUpdated'
+        comment:
           type: string
-        name:
+          readOnly: true
+        source:
           type: string
+          readOnly: true
         id:
           type: string
         _projectId:
           type: string
+          readOnly: true
+        _refId:
+          type: string
+          readOnly: true
         _creator:
           type: string
+          readOnly: true
         _docId:
           type: string
-        _modifier:
+          readOnly: true
+        _created:
           type: string
-        _modified:
+          readOnly: true
+        _commitId:
           type: string
-        _created:
+          readOnly: true
+      additionalProperties: true
+    CommitsRequest:
+      required:
+      - commits
+      type: object
+      properties:
+        commits:
+          type: array
+          items:
+            $ref: '#/components/schemas/Commit'
+    ElementUpdated:
```

Comment:
I think `ElementVersion` (consistent with SysML v2), `ElementChange`, `ElementRevision`, etc. may be a better name as update is usually connoted as disjoint from create/delete (CRUD).

Reply:
changed to ElementVersion